spec/02-integration/05-proxy/10-balancer/03-consistent-hashing_spec.lua

```diff
@@ -50,10 +47,6 @@ for _, strategy in helpers.each_strategy() do
           local api_host = bu.add_api(bp, upstream_name)
           bu.end_testcase_setup(strategy, bp)
 
-          if strategy ~= "off" then
-            helpers.wait_for_all_config_update()
```

Comment:
I am curious why this is no longer needed.

Reply:
Please see https://github.com/Kong/kong/blob/7397d6ffdbed91ccf2d6077dc80d992d162bb684/spec/fixtures/balancer_utils.lua#L482-L509